cmd/queue/main.go

```diff
@@ -287,33 +247,35 @@ func main() {
 		os.Exit(0)
 	}
 
+	// Parse the environment.
 	var env config
 	if err := envconfig.Process("", &env); err != nil {
 		fmt.Fprintln(os.Stderr, err)
 		os.Exit(1)
 	}
 
+	// Setup the logger.
 	logger, _ = logging.NewLogger(env.ServingLoggingConfig, env.ServingLoggingLevel)
 	logger = logger.Named("queueproxy")
 	defer flush(logger)
 
-	initConfig(env)
 	logger = logger.With(
-		zap.String(logkey.Key, servingRevisionKey),
+		zap.String(logkey.Key, types.NamespacedName{Namespace: env.ServingNamespace, Name: env.ServingRevision}.String()),
```

Comment:
How does this actually change logs? I am afraid the key might look like  `{a:b,c:d}`, which isn't very readable.

Reply:
Nope, the stringified version (note the `.String()` is the same as before.